pyonfx/ass_core.py

```diff
@@ -138,6 +138,10 @@ class Char:
         postspace (int): Char free space after text.
         width (float): Char text width.
         height (float): Char text height.
+        ascent (float): Line font ascent (*).
```

Comment:
All these comments could be adjusted: these fields are not line-releated, but font-related. Probably "Font's ascent" is enough.